pytensor/tensor/basic.py

```diff
@@ -1468,10 +1474,21 @@ def make_node(self, value, *shape):
         otype = TensorType(dtype=value.dtype, shape=combined_static_shape)
         return Apply(self, [value] + shape, [otype()])
 
+    @staticmethod
+    def _check_runtime_broadcast(node, value, shape):
```

Comment:
How does this interact with jax jit? Is this code executed every time function runs or at the creation time?